Adds task creation functionality for the "Admin" role.

ArseniyVasilko opened the pull request:
Adds task creation functionality for the "Admin" role.
Once a task is created, it is automatically assigned to each existing group and tracked individually.

Logeshsharma reviewed (commented):
Implementation - lgtm

kind add nav bar - navigation

ArseniyVasilko committed:
Adds task creation functionality for the "Admin" role.
Once a task is created, it is automatically assigned to each existing group and tracked individually.

ArseniyVasilko committed:
Additional start_datetime, end_datetime and location fields added to the db model.
Same fields and respective validation added to the task form as well.
"Create A Task" link added to navbar (Admins only)

Logeshsharma commented:
lgtm, well done!
